test/methods.js

```diff
@@ -1555,7 +1555,7 @@ QUnit.test( "cifES", function( assert ) {
 	assert.ok( method( "S5141387J" ), "CIF valid" );
 	assert.ok( method( "S9803881C" ), "CIF valid" );
 	assert.ok( !method( "S98038813" ), "CIF invalid: digit control must be a letter (C)" );
-	assert.ok( method( "s98038813" ), "CIF valid: lower case" );
+	assert.ok( !method( "s98038813" ), "CIF invalid: lower case but digit control must be a letter (C)" );
```

Comment:
why is this considered invalid now?

in your comment you said lower case chars are fine.
what das "but digit control must be a letter" mean?